custom/icds_reports/tasks.py

```diff
@@ -338,13 +355,15 @@ def _aggregate_df_forms(state_id, day):
 
 
 @track_time
-def _aggregate_child_health_pnc_forms(state_id, day):
+def _aggregate_pnc_forms(state_id, day):
     AggregateChildHealthPostnatalCareForms.aggregate(state_id, day)
+    AggregateCcsRecordPostnatalCareForms.aggregate(state_id, day)
 
 
 @track_time
 def _aggregate_thr_forms(state_id, day):
     AggregateChildHealthTHRForms.aggregate(state_id, day)
+    AggregateCcsRecordTHRForms.aggregate(state_id, day)
```

Comment:
Might be nice to keep these separate. There shouldn't be any dependencies between their generation and it would enable having separate ccs record and child health aggregate workflows